wdn/templates_6.0/examples/card.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dialog</title>
+    <link rel="stylesheet" href="/wdn/templates_6.0/css/critical.css" />
+    <link rel="stylesheet" href="/wdn/templates_6.0/css/main.css" />
+    <script src="/wdn/templates_6.0/js/head-2.js"></script>
+  </head>
+  <body class="unl">
+    <main>
+      <h1>Card</h1>
+      <div
+        id="test-card"
+        class="unl-bg-cream dcf-card dcf-d-flex dcf-flex-col unl-box-shadow unlcms-component-spacer dcf-card-as-link"
```

Comment:
I would change this markup to not be so CMS focused. There is a good example in the `./debug.shtml` file